### DISCUSSION TITLE
How to specify ALPN protocol?

### DISCUSSION THREAD

**Discussion by ttc0419** [curl/curl#10183] [Q&A] [answered]
Hi, I want curl to send http 1.0 as the ALPN protocol.  How can I do that?  If I use the option ```--http1.0```, it will send http 1.0 request after TLS handshake.  But the ALPN protocol is still http 1.1 (```* ALPN: offers http/1.1```).

**Comment by bagder** [comment, marked as answer]
I think that's a bug. There's no separate option to specify the ALPN id.

**Reply by ttc0419** [reply]
Thanks for the reply, should I open an issue for this?

**Reply by bagder** [reply]
Sure, do that. I'll work on a PR...

**Reply by bagder** [reply]
See #10184
